front/src/common/api/generatedEditoastApi.ts

```diff
@@ -3082,6 +3160,34 @@ export type TrainScheduleResult = TrainScheduleBase & {
   id: number;
   timetable_id: number;
 };
+export type TowedRollingStock = {
+  comfort_acceleration: number;
+  gamma: Gamma;
+  id: number;
+  inertia_coefficient: number;
+  length: number;
+  locked: boolean;
+  mass: number;
+  name: string;
+  railjson_version: string;
+  rolling_resistance: RollingResistance;
+  startup_acceleration: number;
+};
+export type TowedRollingStockForm = {
+  comfort_acceleration: number;
+  gamma: Gamma;
+  inertia_coefficient: number;
+  length: number;
+  locked: boolean;
+  mass: number;
+  name: string;
+  rolling_resistance: RollingResistance;
+  startup_acceleration: number;
+};
```

Comment:
Instead of having two types almost identical, could this one have been an Omit<TowedRollingStock, ["id", "railjson_version"]> ? (in rust equivalent)

Reply:
I do not think this is possible in Rust (or at least, I don't see an obvious solution). Note that I mostly reproduce what already exist on `RollingStock` and `RollingStockForm` (but that is not a good argument to not look for improvement, noted 😄).

If I understand correctly, it's some kind of inheritance. The OpenApi format does support inheritance, but I looked for inheritance in `utoipa` (the tool in `editoast` that generates OpenApi) without finding anything.

Reply:
Yes you're right it's not the only spot where it could be changed. Was just to pointing that out but I understand it might be difficult to target that behavior with open api conversion :) We can keep that in mind for later if we find a better solution.

Reply:
That said, in both cases, it's 2 different types, so writing it with `Omit` or in full version is the same if I understand well. And we could argue it's more code to maintain... but this code is auto-generated, so it doesn't even apply here.